src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleHost.cs

```diff
@@ -2916,7 +2916,7 @@ private class ConsoleHostStartupException : Exception
             {
             }
 
-            protected
+            private
             ConsoleHostStartupException(
```

Comment:
Please fix this in another PR.

Reply:
It is a compiler error, it must be fixed here.

Reply:
Because of `sealed`?

Reply:
> A sealed class cannot introduce a protected member because no other class will be able to inherit from the sealed class and use the protected member.

https://docs.microsoft.com/en-gb/dotnet/csharp/misc/cs0628